Gaijinizer patch/Map459.txt

```diff
@@ -3,660 +3,795 @@
 
 > BEGIN STRING
 あるいは、さっきのブラックドッグと同じで
 \\C[2]刺客の誰かに連れてこられたのか……\\C[0]
 
 可能性としては、そっちの方が高いね。
-> CONTEXT: Map459/events/5/pages/0/61/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map459/events/5/pages/0/61/Dialogue/sirousagi_fc1
+They could have been \\C[2]brought here by one of the
+assassins\\C[0] like the Black Dogs were instead...
 
+That seems to be more likely.#Amaboye
 > END STRING
 
 > BEGIN STRING
 でも、油断しないで
 あいつはキメラ種の中でも、完成された個体……
 
 少しでも隙を見せたら、すぐにヤられるよ。
-> CONTEXT: Map459/events/5/pages/0/66/Dialogue/sirousagi_fc1 < UNTRANSLATED
+> CONTEXT: Map459/events/5/pages/0/66/Dialogue/sirousagi_fc1
+But don't let your guard down...
 
+This is a fully-realized chimera we're dealing
+with, she'll take any opening you give her.#Amaboye
 > END STRING
 
 > BEGIN STRING
 "EX4　キメラビースト"
-> CONTEXT: Map459/events/5/pages/0/74/InlineScript/2:1 < UNTRANSLATED
-> CONTEXT: Map460/events/154/pages/0/22/InlineScript/2:1 < UNTRANSLATED
-
+> CONTEXT: Map459/events/5/pages/0/74/InlineScript/2:1
+> CONTEXT: Map460/events/154/pages/0/22/InlineScript/2:1
+"EX4 Chimera Beast"#Amaboye
 > END STRING
 
 > BEGIN STRING
 ………搾精目標、補足……
 
 陵辱準備……開始……
-> CONTEXT: Map459/events/5/pages/0/80/Dialogue/c_beast_fc1 < UNTRANSLATED
-
+> CONTEXT: Map459/events/5/pages/0/80/Dialogue/c_beast_fc1
+...Adding targets to list to be squeezed...
+Initializing rape program...#Amaboye
 > END STRING
 
 > BEGIN STRING
 来るよ、\\N[1]くん！！
-> CONTEXT: Map459/events/5/pages/0/84/Dialogue/enemy30 < UNTRANSLATED
-
+> CONTEXT: Map459/events/5/pages/0/84/Dialogue/enemy30
+\\N[1]-kun, here they come!#Amaboye
 > END STRING
 
 > BEGIN STRING
 ………行動、不能……
 
 一時……退却………
-> CONTEXT: Map459/events/5/pages/0/102/Dialogue/c_beast_fc1 < UNTRANSLATED
-
+> CONTEXT: Map459/events/5/pages/0/102/Dialogue/c_beast_fc1
+...Continued action, impossible...
+Temporary... retreat...#Amaboye
 > END STRING
 
 > BEGIN STRING
 き、キミ…！！大丈夫か！？
-> CONTEXT: Map459/events/5/pages/0/141/Dialogue/kazuya < UNTRANSLATED
-
+> CONTEXT: Map459/events/5/pages/0/141/Dialogue/kazuya
+H-hey...! Are you alright!?#Amaboye
 > END STRING
 
 > BEGIN STRING
 学生「あ……あへ……♡あぁ……
 
 　　　も、もう出な……♡」
-> CONTEXT: Map459/events/5/pages/0/146/Dialogue < UNTRANSLATED
-
+> CONTEXT: Map459/events/5/pages/0/146/Dialogue
+Student- 「Ah... ahe...♡ Aah...
+           N... no more...♡」#Amaboye
 > END STRING
 
 > BEGIN STRING
 …………よかった、まだ１回か２回だったみたいだ……
 
 大した衰弱具合じゃなさそう……
-> CONTEXT: Map459/events/5/pages/0/150/Dialogue/kazuya < UNTRANSLATED
+> CONTEXT: Map459/events/5/pages/0/150/Dialogue/kazuya
+...Thank goodness, it seems he only came once or
+twice...
 
+He doesn't seem too weakened...#Amaboye
 > END STRING
 
 > BEGIN STRING
 すごい身体ビクビクしてるけど
 吸われたのはギリギリ精気だけっぽいから
 
 魂も吸収されてないし、ひとまず安心だね！
-> CONTEXT: Map459/events/5/pages/0/154/Dialogue/enemy15 < UNTRANSLATED
-
+> CONTEXT: Map459/events/5/pages/0/154/Dialogue/enemy15
+His body's shaking like crazy, but it seems she
+just barely stopped at his energy, and his soul
+seems intact, so that's a relief!#Amaboye
 > END STRING
```

Comment:
Splitting this into two sentences for flow:
```
His body's shaking like crazy, but it seem she
just barely stopped at taking his energy.

His soul seems intact, so that's a relief!
```